lua/core/mappings.lua

```diff
@@ -15,15 +14,15 @@ maps.n["Q"] = "<Nop>"
 maps.n["|"] = { "<cmd>vsplit<cr>", desc = "Vertical Split" }
 maps.n["\\"] = { "<cmd>split<cr>", desc = "Horizontal Split" }
 
--- Packer
-maps.n["<leader>pc"] = { "<cmd>PackerCompile<cr>", desc = "Packer Compile" }
-maps.n["<leader>pi"] = { "<cmd>PackerInstall<cr>", desc = "Packer Install" }
-maps.n["<leader>ps"] = { "<cmd>PackerSync<cr>", desc = "Packer Sync" }
-maps.n["<leader>pS"] = { "<cmd>PackerStatus<cr>", desc = "Packer Status" }
-maps.n["<leader>pu"] = { "<cmd>PackerUpdate<cr>", desc = "Packer Update" }
+-- Plugin Manager
+maps.n["<leader>pi"] = { function() require("lazy").install() end, desc = "Plugins Install" }
+maps.n["<leader>ps"] = { function() require("lazy").home() end, desc = "Plugins Status" }
+maps.n["<leader>pS"] = { function() require("lazy").sync() end, desc = "Plugins Sync" }
```

Comment:
these seem to be swapped
```suggestion
maps.n["<leader>pS"] = { function() require("lazy").home() end, desc = "Plugins Status" }
maps.n["<leader>ps"] = { function() require("lazy").sync() end, desc = "Plugins Sync" }
```

Reply:
Yeah I realized the pattern we had before was the opposite of the convention of other lowercase/Capital bindings so that was a fix.